<launch>
  <node name="clean_avoidance_node" pkg="clean_avoidance" type="clean_avoidance_node" output="screen" >
    <rosparam file="$(find clean_avoidance)/param/costmap_common_params.yaml" command="load" ns="global_costmap" />   
    <rosparam file="$(find clean_avoidance)/param/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find clean_avoidance)/param/clean_avoidance.yaml" command="load" />
    </node>
</launch>
